src/core/huff_coder.cc

```diff
@@ -112,6 +110,11 @@ string HuffmanEncoder::Export() const {
   return res;
 }
 
+// Copied from HUF_tightCompressBound.
+size_t HuffmanEncoder::CompressedBound(size_t src_size) const {
+  return ((src_size * num_bits_) >> 3) + 8;
```

Comment:
Haven't seen `tightCompressBound` but why not use the macro here ?

Reply:
why macro?

Reply:
Because I thought it was a macro. My point is, if there is such function or macro, you could use that instead